packages/studio-web/project.json

```diff
@@ -78,7 +78,8 @@
           "localize": ["es"]
         }
       },
-      "defaultConfiguration": "production"
+      "defaultConfiguration": "production",
+      "dependsOn": ["update-version-file"]
```

Comment:
Does this really work, since `update-version-file` is defined in a different package?